routes/signup.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(transaction, callback) {
     form: {
       format: 'html',
       lang: transaction.locale,
-      newsletters: 'mozilla-foundation',
+      newsletters: transaction.newsletters || 'mozilla-foundation',
```

Comment:
I suspect `transaction.newsletters` is always going to be `undefined`.

We need to add the Hidden name to this array: https://github.com/kyoshino/donate.mozilla.org/blob/1098-newsletter-type/components/signup.jsx#L48